SRC/NETWORK/hget.c

```diff
@@ -174,7 +186,8 @@ byte authenticationSent;
 int remainingInputData = 0;
 byte* inputDataPointer;
 int emptyLineReaded;
-long contentLength;
+long contentLength,blockSize,currentBlock;
+bool bFirstUpdate;

```

Comment:
Ok, I understand that hungarian notation is commonly used in the world of C, but since I haven't used it for this program I'd prefer you don't use it either, to keep the code consistenct. Here are suggestions for alternative names:

`bFirstUpdate` -> `isFirstUpdate`
`bTls` -> `tlsIsSupported`
`bHTTPS` -> `useHttps`
`bCheckCertificate` -> `mustCheckCertificate`
`bCheckHostName` -> `mustCheckHostName`
`bSafeTLSSupport` -> `safeTlsIsSupported`

Reply:
Done :)